api/src/test/java/com/epam/pipeline/manager/pipeline/RunScheduleManagerTest.java

```diff
@@ -103,214 +117,238 @@ public void setUp() {
         testPipeline.setRepository(TEST_REPOSITORY);
         testPipeline.setOwner(TEST_NAME);
         testPipeline.setRepositorySsh(TEST_PIPELINE_REPO_SSH);
-        pipelineDao.createPipeline(testPipeline);
+        testPipeline.setId(RUN_ID);
 
         createPipelineRun(RUN_ID, testPipeline.getId());
```

Comment:
You can get rid of this call. It doesn't have any effect now because everything is mock.